ts/Series/Lollipop/LollipopPointOptions.d.ts

```diff
@@ -14,15 +14,22 @@
  *
  * */
 
-import type DumbbellPointOptions from '../Dumbbell/DumbbellPointOptions';
+import type ScatterPointOptions from '../Scatter/ScatterPointOptions';
+import type ColorType from '../../Core/Color/ColorType';
+import DashStyleValue from '../../Core/Renderer/DashStyleValue';
```

Comment:
Should be `import type`?

Reply:
I was looking for similar files with that kind of importing options and we are using `import type`.

Reply:
Yes, I mean it should be `import type DashStyleValue` too?

Reply:
Oh, didn't notice. Thank you for the clarification!